src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -0,0 +1,202 @@
+For issues that the primary does not know how to resolve, the backup should be
+enlisted. It may just be a quick response that the backup can answer. If the
+backup cannot help, the primary should enlist other developers using the @
+mention feature of the associated GitHub issue. However, where a primary is
+responsible for maintaining a one-hour response time, other VisIt_ developers
+so enlisted are free to either delay or even decline to respond (but nonetheless
+inform the primary of this need) if their current schedules do not permit
+timely response.
+
+If the work required to resolve a customer response incident is either not
+known or not believed to be a *low-hanging-fruit* type task, the primary should
+search the issue system to see if this is a known issue and, if so, add
+additional information to that known issue about this new customer response
+incident (and perhaps remove the *reviewed* tag from the issue to cause the
+issue to be re-reviewed at the next VisIt_ project meeting) or submit a *new*
+issue to the main repository issues.
+
+Special Considerations for SCF
+------------------------------
+
+Occasionally, incidents arise that may be handled only on the SCF. This is
+not too common but does happen and it presents problems for a geographically
+distributed team.
+
+On the one hand, customers on SCF are accustomed to longer response times.
+On the other hand, often work on the SCF is a high priority and requires
+rapid response from a developer that is on site with access to SCF.
+
+In many ways, an SCF-only incident is just a different form of *escalation*.
+
+Our current plan is to handle this on a case-by-case basis. If neither the
+primary nor backup are able to handle a customer response incident requiring
+the SCF, the primary should
+
+  * First determine the customer's required response time. It may be hours
+    or it may be days. If it is days. Its conceivable the issue could be
+    handled in the following week by a new primary/backup pair.
+  * If customer indicates immediate response required, primary should inquire
+    the whole team to arrange another developer who can handle it.
+
+Relation to Programmatic Work
+-----------------------------
+
+When developers are faced with many development tasks, its easy to think of
+customer response work as an *interruption* to those tasks. However, customer
+response *is* indeed a critical aspect to programmatic work. It is part of what
+is involved in keeping VisIt_ a working and useful tool in our customer's
+workflows not only here at LLNL, though that is VisIt_'s biggest customer, but
+wherever in DOE/DOD and elsewhere in the world VisIt_ is used. Often the issues
+users at large encounter and report are relevant to the use of VisIt_ here at
+LLNL.
+
+To Do
+-----
+
+  * Decide when to switch to this new process
+  * Where do we keep weekly "calendar"?
+    * maybe a wiki page or issue on live issues repo?
```

Comment:
I think Outlook would be a good tool for this because it has an actual, visual calendar. We can make an account (e.g. visit-developers@doellnl.onmicrosoft.com). This account will have its own calendar where we can track and assign customer response weeks via calendar events.

Reply:
Not everyone uses Outlook. Seems a bit more complicated than necessary. I started a [dummy wiki page](https://github.com/visit-dav/live-customer-response/wiki/VisIt-Customer-Response-Primary-and-Backup-Schedule) as an example.

Reply:
Doesn't necessarily have to be Outlook, could be anything with a calendar. I see what you mean about the chart being easier to list. However, I do wonder how easy *planning* will be with just a chart in the midst of time-splits, conferences, vacations, etc. Maybe we don't need a calendar but a table instead where each person can indicate which weeks they can work on Visit... Something like:
```
+ ------ + ----- + ---- + ---- +
|        | Eddie | Mark | Eric |
+ ------ + ----- + ---- + ---- +
| Feb 3  | Busy  | Free | Busy |
| Feb 10 | Free  | Busy | Free |
| Feb 17 | Free  | Free | Busy |
+ ---- + ----- + ---- + ---- +
```
Then from table we can assign duties for the quarter.